docs/api/en/core/BufferGeometry.html

```diff
@@ -297,20 +297,26 @@ <h3>[method:this rotateX] ( [param:Float radians] )</h3>
 			Rotate the geometry about the X axis. This is typically done as a one time
 			operation, and not during a loop. Use [page:Object3D.rotation] for typical
 			real-time mesh rotation.
+
+			Rotates the object around x axis in world space.
```

Comment:
TBH, this new line is confusing since `BufferGeometry` just represents geometry data and coordinate spaces are not relevant there.